inst/text/gitignore.txt

```diff
@@ -23,10 +37,43 @@ data/
 *.[rR][dD][aA][tT][aA]
 *.[rR][dD][sS]
 
-# Cross platform data files #
+# Cross-platform data files #
 *.[pP][aA][rR][qQ][uU][eE][tT]
 *.[fF][sS][tT]
 *.[qQ][sS]
 
 # MacOS folder attributes files #
 .DS_Store
+
+# produced vignettes
+vignettes/*.html
+vignettes/*.pdf
+
+# OAuth2 token, see https://github.com/hadley/httr/releases/tag/v0.3
+.httr-oauth
+
+# knitr and R markdown default cache directories
+*_cache/
+/cache/
+
+# Temporary files created by R markdown
+*.utf8.md
+*.knit.md
+
+# pkgdown site
+docs/
```

Comment:
Also may be specific to building R packages rather than R projects generally?

Reply:
Yep, as in the other comment. I can remove the package-specific bits if you want but my thoughts were that it doesn't hurt anyone to have them in...